framework/aster-frame/Cargo.toml

```diff
@@ -31,15 +31,15 @@ spin = "0.9.4"
 static_assertions = "1.1.0"
 tdx-guest = { version = "0.1.0", optional = true }
 trapframe = { git = "https://github.com/asterinas/trapframe-rs", rev = "4739428" }
-unwinding = { version = "0.2.1", default-features = false, features = ["fde-gnu-eh-frame-hdr", "hide-trace", "panic", "personality", "unwinder"] }
+unwinding = { git = "https://github.com/nbdd0121/unwinding.git", rev = "d7cd46e", default-features = false, features = ["fde-gnu-eh-frame-hdr", "hide-trace", "panic", "personality", "unwinder"] }
```

Comment:
Why switch to Gary Guo's version of unwinding? Explain the reason with a comment so that we know when is the right time to switch back to the upstream version.